MotorRandomWee/MotorRandomWee.ino

```diff
@@ -78,8 +81,8 @@ int getPIDSpeed ()
   Serial.print (imuZAvg);
```

Comment:
Can the serial.print stuff be in a separate debug function?

Reply:
nah